internal/matomo/matomo.go

```diff
@@ -0,0 +1,119 @@
+package matomo
+
+import (
+	"crypto/rand"
+	"fmt"
+	"math/big"
+	"net/http"
+	"net/url"
+	"os"
+	"runtime"
+	"strings"
+	"time"
+
+	"github.com/scaleway/scaleway-cli/internal/terminal"
+	"github.com/scaleway/scaleway-sdk-go/logger"
+	"github.com/scaleway/scaleway-sdk-go/scw"
+)
+
+const (
+	// full reference API https://developer.matomo.org/api-reference/tracking-api
+	endpoint     = "kindermoumoute.matomo.cloud/matomo.php"
+	project      = "4"
+	enableRecord = "1"
+	apiVersion   = "1"
+)
+
+var ForceTracking = os.Getenv("SCW_FORCE_TRACKING") == "true"
+
+type TrackCommandRequest struct {
+	RunCommand    string
+	Version       string
+	ExecutionTime time.Duration
+}
+
+func TrackCommand(request *TrackCommandRequest) error {
+	// disable tracking when no command was run, or on completion command
+	if request.RunCommand == "" || strings.HasPrefix(request.RunCommand, "autocomplete complete") {
+		return nil
+	}
+
+	// compute or retrieve tracking parameters
+	terminalResolution := fmt.Sprintf("%dx%d", terminal.GetWidth(), terminal.GetHeight())
+	commandDurationInMs := fmt.Sprintf("%d", request.ExecutionTime/time.Millisecond)
+	randNumber := generateRandNumber()
+	action, actionURL := commandToAction(request.RunCommand)
+	userAgent := fakeUserAgent(request.Version)
```

Comment:
what's the difference with a true `userAgent` ?

Reply:
The true user agent(in the SDK) is not exposed publicly